src/main/java/delete_post/use_case/DeletePostRequestModel.java

```diff
@@ -2,31 +2,23 @@
 
 import entities.CurrentUser;
 import entities.User;
-import entities.Post;
 
 public class DeletePostRequestModel {
-    private Post post;
+    int postId;
     private boolean isAdmin = CurrentUser.getIsAdmin();
     private User currentUser = CurrentUser.getCurrentUser();
```

Comment:
I don't think the currentuser stuff needs to be stored here, as it can be accessed directly from the interactor